nitransforms/nonlinear.py

```diff
@@ -22,9 +31,24 @@ class DisplacementsFieldTransform(TransformBase):
     __slots__ = ["_field"]
 
     def __init__(self, field, reference=None):
-        """Create a dense deformation field transform."""
+        """
+        Create a dense deformation field transform.
+
+        Example
+        -------
+        >>> DisplacementsFieldTransform(test_dir / "someones_displacement_field.nii.gz")
+        <(57, 67, 56) field of 3D displacements>
+
+        """
         super().__init__()
-        self._field = np.asanyarray(field.dataobj)
+
+        field = _ensure_image(field)
+        self._field = np.squeeze(
+            np.asanyarray(field.dataobj) if hasattr(field, "dataobj") else field
+        )
```

Comment:
No need to change anything, but I think `_ensure_image` doesn't quite capture the operation of something that will pass ndarrays through untouched.

Reply:
Is this a comment on the function's name, the function's behavior, or both? 
Happy to move towards something more appropriate.